CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+### Features
+
+- Allow embedding SQL query bindings (parameters) in SQL query breadcrumbs and spans.
+
+  To enable this feature, update your `config/sentry.php` file or set the `SENTRY_TRACE_SQL_BINDINGS_ENABLED` and `SENTRY_BREADCRUMBS_SQL_BINDINGS_ENABLED` environment variables to `embed`.
```

Comment:
I considered the addition of an additional boolean configuration option for this, but found that this complicated the code and made it unclear which option had precedence over the existing options. However, I could see reverting back to making the options just a boolean and simply adopting the new behaviour?